src/index.html

```diff
@@ -7,8 +7,72 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <a href="#">
+        <img
+          class="logo"
+          src="../src/images/logo.png"
+          alt="logo">
+      </a>
+      <nav class="navbar">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              class="nav_link is-active" href="#">
+            APPLE
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a class="nav_link" href="#">
+            SAMSUNG
+            </a>
```

Comment:
```suggestion
            <a class="nav_link" href="#">
              SAMSUNG
            </a>
```
fix code style all over the page